config/clusters/cloudbank/demo.values.yaml

```diff
@@ -42,15 +42,11 @@ jupyterhub:
           http://google.com/accounts/o8/id:
             username_derivation:
               username_claim: "email"
-            # allow_all is a partial authorization, username_pattern is enforced also
-            allow_all: true
-          urn:mace:incommon:berkeley.edu:
-            username_derivation:
-              username_claim: "email"
-            # allow_all is a partial authorization, username_pattern is enforced also
+            # allow_all is a partial authorization for this hub because
+            # username_pattern configured and enforced also, allowing only users
+            # with *.edu suffixed domains besides specific admin users.
             allow_all: true
```

Comment:
This is now the only situation `username_pattern` is adopted, and I'd like us to minimize this to avoid complexity and rely instead on more establishes practices whenever possible.

In this case, we wanted to allow a range of domains suffixied with `*.edu` and for that we have no tool.

Reply:
OK, I think we just lock demo down to just us and when people want to use it I can add their Gmail address to allowed_users temporarily. The only caveat would be that in June for the Data Science National Conference, we would want to open it up for that week

Reply:
@sean-morris okay! I've commented out the username_pattern and made sure that its easy to enable back again for you temporarily in the future.

Reply:
Sounds great.